Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -58,24 +75,40 @@ private let routers: [Router<Route>] = [
   Route.iso.about
     <¢> get %> lit("about") <% end,
 
-  Route.iso.account
+  Route.iso.account <<< Route.Account.iso.index
     <¢> get %> lit("account") <% end,
 
+  Route.iso.account <<< Route.Account.iso.paymentInfo <<< Route.Account.PaymentInfo.iso.show
+    <¢> get %> lit("account") %> lit("payment-info") <% end,
+
+  Route.iso.account <<< Route.Account.iso.paymentInfo <<< Route.Account.PaymentInfo.iso.update
```

Comment:
just texted this to you, but wonder if we could beef up our sourcery template so that we can just do `.account <<< .paymentInfo <<< .update`